config/initializers/voyager_helpers.rb

```diff
@@ -1,5 +1,4 @@
 require 'yaml'
-require_relative '../../voyager_helpers/lib/voyager_helpers'
 include VoyagerHelpers::OracleConnection
 
 vh_config = YAML.load_file("#{File.dirname(__FILE__)}/../voyager_helpers.yml")
```

Comment:
It does feel a little convoluted ([this](https://github.com/pulibrary/marc_liberation/blob/development/config/initializers/voyager_helpers.rb)), though I suppose it's a valid approach ([I say since I did it](https://github.com/pulibrary/marc_liberation/blame/development/config/initializers/voyager_helpers.rb) :frowning: ). I've never really known which was better:

 1. Put that kind of config data in YAML so that (in a perfect world) the sysadmin deploying the app doesn't have to touch Ruby
 2. Put it in Ruby. Sysadmin deals.
 3. The application assumes some environment variables.

I'm OK w/ any of the above, but we should be consistent.

I think keeping the instructions somewhat vague in voyager_helpers is fine, since not everyone using it will be in Rails land.